### PR TITLE
some tweaks for Subview

### PR DESCRIPTION
**Pull request by Mariusthvdb**
and take out the 'more info below' since there was no more info below. Add a basic subview example.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/dashboards/views.markdown [hunk 1]
```diff
@@ -221,3 +221,17 @@ View configuration:
...
       image: /local/lights.png
 ```
+
+Subview configuration:
+
+```yaml
+title: Energieprijzen
+path: energieprijzen
+subview: true
+back_path: /ui-data/climate
```
Comment (frenck) [inline source/dashboards/views.markdown]:
```suggestion
title: "Energieprijzen"
path: "energieprijzen"
subview: true
back_path: "/ui-data/climate"
```

File: source/dashboards/views.markdown [hunk 2]
```diff
@@ -221,3 +221,17 @@ View configuration:
...
+
+cards:
+  - type: entities
+    entities:
+      - etc
```
Comment (frenck) [inline source/dashboards/views.markdown]: etc?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/dashboards/views.markdown (hunk 2 above)
Comment (Mariusthvdb) [reply source/dashboards/views.markdown]: yeah, well, I guess that would be obvious. you want me to take out that line 236, or, prefer an entity
 like `sensor.today_avg_price`

Comment (frenck) [reply source/dashboards/views.markdown]: Add some example, that would help :)

Comment (Mariusthvdb) [reply source/dashboards/views.markdown]: done

**Review by frenck — APPROVED** [review]
Thanks, @Mariusthvdb 👍

**Pull request merged**